src/main/java/com/iota/iri/model/persistables/Transaction.java

```diff
@@ -9,6 +9,7 @@
 import java.nio.ByteBuffer;
 
 public class Transaction implements Persistable {
+    private static final long serialVersionUID = 113808742723953354L;
```

Comment:
What is this for? To identify changes in the serialization in the future? How do you expect de s to use it?

Reply:
It was auto-implemented by the IDE (VS code), and gave warnings otherwise. Apparently it's a Java standard for classes implementing `Serializable` (see [here](https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it)). I'm not sure how `Persistable` fits in the story but I went with the recommendation to get rid of the warning.